modules/social_features/social_event/social_event.install

```diff
@@ -2163,3 +2163,35 @@ function social_event_update_11403(): string {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Add new "Hide enrollments" field to Event node type.
+ */
+function social_event_update_11501(): void {
+  $config_path = \Drupal::service('extension.list.module')->getPath('social_event') . '/config/static';
+  $source = new FileStorage($config_path);
+
+  // Create a new field from the yaml configuration and save.
+  \Drupal::entityTypeManager()->getStorage('field_storage_config')
+    ->create((array) $source->read('field.storage.node.field_hide_enrollments_11501'))
```

Comment:
Why we have chosen to type caste it as array ?

Reply:
Because otherwise we get `phpstan` checks failed.